src/payment/scripts/db-schema.sql

```diff
@@ -1 +1,55 @@
-CREATE TABLE users
+CREATE TABLE tUser (
+	UserID serial PRIMARY KEY,
+	LastName VARCHAR(64) NOT NULL,
+	FirstName VARCHAR(64) NOT NULL,
+	Address VARCHAR(255),
+	City VARCHAR(255),
+	Email VARCHAR(255) UNIQUE NOT NULL,
+	CreditCard VARCHAR(40),
+	created_on TIMESTAMP NOT NULL
```

Comment:
Is the `created_on` timestamp needed for every table?

Reply:
definitely user and reservation table, let me think about the others